packages/rangelink-vscode-extension/src/__tests__/matchers/toBeRangeLinkExtensionError.test.ts

```
@@ -0,0 +1,241 @@
+import { ExtensionResult } from '../../types/ExtensionResult';
```

ESLint

[error] `../../types/ExtensionResult` import should occur after import of `../../errors/RangeLinkExtensionErrorCodes`

(import/order)